src/session.rs

```diff
@@ -166,6 +166,61 @@ impl Session {
         )
     }
 
+    /// Constructs a new [`Command`] for launching subsystem `program` on the remote
+    /// host.
+    ///
+    /// Unlike [`command`](Session::command), this method does not shell-escape `program`, so it may be evaluated in
+    /// unforeseen ways by the remote shell.
```

Comment:
Why not?

Reply:
I think in sshd, subsystem invocation are treated different from normal command, in that it may not be passed to shell at all.

For example, if the sftp-internal subsystem is enabled, then the sftp server is run directly inside sshd process.